source/common/common/utility.h

```diff
@@ -254,6 +258,15 @@ class StringUtil {
    * @return std::string s converted to upper case.
    */
   static std::string toUpper(absl::string_view s);
+
+  /**
+   * Removes all the character indices from str contained in the interval-set.
+   * @param str the string containing the characters to be removed.
+   * @param remove_characters the set of character-intervals .
```

Comment:
typo " ."

Reply:
done.

@mrice32 I added what I think is a complete coverage of corner cases based on a starting condition of three intervals with some room in between.